package.json

```diff
@@ -28,7 +27,7 @@
     "promise": "8.0.1",
     "raf": "3.4.0",
     "react": "^16.2.0",
-    "react-dev-utils": ">=4.2.2",
+    "react-dev-utils": "4.2.1",
```

Comment:
This version bump caused issues with the template boilerplate that I did not debug, so I'm simply reverting the change.